README.md

```diff
@@ -1,3 +1,7 @@
 # Litee-gh-testing
 
 this is a test repository for developing `litee-gh.nvim`
+
+woops
```

Comment:
ok this time I did `GHStartReview` first

Reply:
This was posted about the cool stuff, not the woopsies

Reply:
So in gh.nvim you created this comment on line 7?